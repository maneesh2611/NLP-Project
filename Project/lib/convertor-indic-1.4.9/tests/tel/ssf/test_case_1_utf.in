<Sentence id="1">
1	రాష్ట్ర	unk	
2	పర్యటనకు	unk	
3	వస్తున్న	unk	
4	శ్రీకృష్ణ	unk	
5	కమిటీకి	unk	
6	సమర్పించేందుకు	unk	
7	టీజీ	unk	
8	వెంకటేష్	unk	
9	నివేదికను	unk	
10	సిద్దం	unk	
11	చేసారు	unk	
12	.	unk	
</Sentence>
