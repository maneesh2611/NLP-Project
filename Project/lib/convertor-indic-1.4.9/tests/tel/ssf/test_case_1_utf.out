<Sentence id="1">			
1	rARtra	unk	
2	paryatanaku	unk	
3	vaswunna	unk	
4	SrIkqRNa	unk	
5	kamitIki	unk	
6	samarpiMceMxuku	unk	
7	tIjI	unk	
8	veVMkateR	unk	
9	nivexikanu	unk	
10	sixxaM	unk	
11	cesAru	unk	
12	.	unk	
</Sentence>			
